### PR TITLE
content module schema returns hrefs for anchor tags

### PR DESCRIPTION
**Pull request by lionsdwarf**

### CODE REVIEW

File: apps/api/models/content-module.js [hunk 1]
```diff
@@ -93,11 +94,20 @@ export const airtableToModel = record => {
       return { tag: $(el).get(0).tagName, text: $(el).text() }
     })
     .get()
-  const pWrappedEls = $('body > p > *')
+  let pWrappedEls = $('body > p > *')
     .map((i, el) => {
-      return { tag: $(el).get(0).tagName, text: $(el).text() }
+      const tagObj = { tag: $(el).get(0).tagName, text: $(el).text() }
+      // If, as below, we extract more metadata from markdown elements we should refactor the return types to objects instead of strings
```
Comment (craigspaeth) [inline apps/api/models/content-module.js]: I'd love to keep character lengths under 80 characters (enforced by the linter on code and generally good [legibility typography convention](https://baymard.com/blog/line-length-readability)). Mind breaking this comment onto multiple lines around the `markdown elements we` bit?

File: apps/api/models/content-module.js [hunk 2]
```diff
@@ -93,11 +94,19 @@ export const airtableToModel = record => {
...
+      // Plucking out nested hrefs:
+      return el.attribs.href
+        ? [tagObj, { tag: 'href', text: el.attribs.href }]
+        : tagObj
     })
     .get()
+  // If hrefs were plucked, pWrappedEls will contain arrays of tag objects
+  // In this cases, flatten the array of arrays:
+  pWrappedEls = [].concat.apply([], pWrappedEls)
```
Comment (craigspaeth) [inline apps/api/models/content-module.js]: FWIW [Lodash](https://github.com/findOctave/cortex/blob/a608c0158cf7eb9f0441405e6e45ca7a1d78305c/apps/api/models/content-module.js#L45) has a nice function for this `_.flatten(pWrappedEls)`

**Comment by craigspaeth** [comment]
Mostly LGTM! To get comfortable with the test suite, I'd love if you could write a test for this before we merge. You can see the [suite here](https://github.com/findOctave/cortex/blob/master/apps/api/test/models/content-modules.test.js#L7) and hopefully, it's straightforward to copy one of those tests and run `npm run jest apps/api/test/models/content-modules.test.js` to run just that test suite. Let me know if you get stuck and want to pair through it. Thanks 🙌

**Comment by craigspaeth** [comment]
Thanks for the test. LGTM and 💚

File: apps/api/models/content-module.js (hunk 1 above)
Comment (lionsdwarf) [reply apps/api/models/content-module.js]: Done.

File: apps/api/models/content-module.js (hunk 2 above)
Comment (lionsdwarf) [reply apps/api/models/content-module.js]: Done

**Comment by lionsdwarf** [comment]
Just updated with test.

**Pull request merged**